scripts/translation/lib/RevcheckFileItem.php

```diff
@@ -62,7 +62,7 @@ public function addGitLogData( string $hash , string $date , bool $skip ) : void
         }
 
         if ( $skip )
-            $this->diffHash = $hash;
+            $this->hashDiff = $hash;
```

Comment:
Good catch. But it got me very worried. I will need to test all translations if this if *actual* results are valid. The silver lining here is that `isSyncHash()` provably is fixing the error, because it is always (?) called after hashes loading.

Reply:
Ok, so let's wait for these tests before merging.

Reply:
Tested on all languages, identical results.